### ISSUE TITLE
Add the possiblity to show the password

### ISSUE THREAD

**Issue by pietervdvn** [openstreetmap/openstreetmap-website#5122]
### Problem

Many users are not sure of their typing skills and would like to see the password while they are typing it. Many websites have a toggle (often an `eye`-icon) which shows the password upon being clicked

### Description

I did a MapComplete-usertest yesterday, and the test subject opened a new tab and typed their password into the URL-bar over there to then copy this back into the password field.

In other words, the security of this is even worse, as it might accidentally send of the password to the search enginge...

**Comment by mmd-osm** [comment]
We've been discussing this topic very recently in #4543 and decided to close it.

**Comment by tordans** [comment]
Btw, it looks like Chrome will have it too "soon", https://www.minitool.com/news/chromium-browsers-reveal-password-001.html

**Comment by HolgerJeromin** [comment]
@tordans 

> Last Updated August 16, 2022

Yes, this was implemented for Microsofts Chromium based browser, but not put upstream. So no other browser has got this feature.
